### ISSUE TITLE
hot pixel?

### ISSUE THREAD

**Issue by nym2834610** [3dem/relion#561]
We recently collected a dataset on ultural-AuFoil grid. After bayesian polishing in RELION3.1, refinement yielded a very low resolution map with wired feature. We ran 2D classification according to previous suggestions to remove particles with hot pixel and ~5% particles showed such features. After removing those particles the refinement is fine. It seems the hot pixel issue will be fixed in 3.1 according to previous posts. I was wondering if there is a way we could get rid of hot pixel without eliminating particles? Thanks a lot.

**Comment by biochem-fan** [comment]
Hot pixels are defects on the detector. If you mean dot-like contaminations in AuFoil grids, they are not hot pixels (they are present on the grid, not the detector). Are you sure you mean hot pixels?

If they are REAL hot pixels, these are two updates in 3.1.
- You can specify the defect pixel map (you or facility manager have to create this). This is used in initial motion correction and Polish.
- The list of hot pixels detected during initial motion correction is stored in the micrograph metadata STAR files and used in Polish.

In other words, you have to run **RELION's** initial motion correction in 3.1, _not 3.0 or UCSF MotionCor2_.

Anyway, my suggestion is not to worry about these particles. Removing 5 % of particles should not change the resolution much if any.

**Comment by nym2834610** [comment]
It seems to be a lot of gold contamination rather than hot pixels. Thanks for the help. However, this raises an issue: we only notice the weird feature in refinements after Bayesian polishing, before which the refinement seems fine. Is there a particular reason?

**Comment by biochem-fan** [comment]
How did you run initial motion correction? In 3.0 or 3.1? With UCSF MotionCor2 or RELION's?
